known-shortcuts/clickup.ts

```diff
@@ -1,3 +1,5 @@
+import { AppParams } from "~/types";
+
 const app = {
```

Comment:
:bug: **Bug**
The `app` object is being cast to `AppParams` using the `as` keyword. While this is valid, it is generally better to define the object with the type directly to ensure that all properties are correctly typed and validated. The `AppParams` type is imported but not used in the code. The `app` object should be explicitly typed as `AppParams` to ensure type safety and to make use of the imported type. This will help in catching any type-related errors during development. Consider using a type annotation instead.

```suggestion
const app: AppParams = {
```